### ISSUE TITLE
I want the scroll to be displayed even when there is no scroll

### ISSUE THREAD

**Issue by krehwell** [kensyo/nvim-scrlbkun#1]
is it possible to make the plugin still active even when my buffer is not scrollable yet?

by doing that I can make sure that I can still see my gitsigns and diagnostic even on a short file (which does not have lots of lines of codes)

**Comment by kensyo** [comment]
No, it's not.

If we comment out lines 203 to 207 in display.lua to display scrlbkun in short files, scrlbkun will look like the following image.
https://github.com/kensyo/nvim-scrlbkun/blob/aa9ef583a05926e754e6406cdcdbe420b85b77c9/lua/scrlbkun/display.lua#L203-L207

<img width="1680" alt="スクリーンショット 2024-02-15 13 17 28" src="https://github.com/kensyo/nvim-scrlbkun/assets/9912146/20cfaa0e-82ca-431b-bc57-0da68e1c7fa2">

On the left side of the screen, we can see the green marks for the new lines by gitsigns, and on the right, by scrlbkun. However, while there is no gap in the green marks by gitsigns, there are some gaps in those by scrlbkun, causing the green marks to be split into multiple parts.
In other words, even though the added lines are continuous, they may not appear to be so in the display by scrlbkun.

Eliminating gaps that occur in short files would require extra implementation, so instead, I decided not to display scrlbkun for short files.
